.github/workflows/ci.yaml

```diff
@@ -141,7 +142,7 @@ jobs:
         - uses: actions/checkout@v2
         - name: Install emirge
           run: |
-            [[ $(uname) == Linux ]] && sudo apt-get install -y openmpi-bin libopenmpi-dev
+            [[ $(uname) == Linux ]] && sudo apt-get update && sudo apt-get install -y openmpi-bin libopenmpi-dev
```

Comment:
No objection here, though maybe there's a benefit to grabbing this from conda-forge?

Reply:
But this is the emirge case - how would we get emirge to use the conda MPI (from a different env)?

Reply:
Fair, I hadn't picked up on that. I was meaning that more for the other CIs, so feel free to disregard this.